src/compiler/factory/nodeFactory.ts

```diff
@@ -281,6 +281,8 @@ namespace ts {
             updateImportEqualsDeclaration,
             createImportDeclaration,
             updateImportDeclaration,
+            createImportCall,

```

Comment:
As with the function itself, this should be placed further down in the list of properties near the `// Compound nodes` comment, since `ImportCall` doesn't have its own `SyntaxKind` and I'd like to keep this list ordered by `SyntaxKind`.